es/basesqlalchemy.py

```diff
@@ -38,10 +38,10 @@ def visit_label(self, *args, **kwargs):
 
 
 class BaseESTypeCompiler(compiler.GenericTypeCompiler):
-    def visit_REAL(self, type_, **kwargs):
+    def visit_REAL(self, type_, **kwargs: Dict[str, Any]) -> str:
```

Comment:
nit: I think `kwargs` should always only type the value, as the key is always `str`
```suggestion
    def visit_REAL(self, type_, **kwargs: Any) -> str:
```